src/formElementsService.ts

```diff
@@ -212,10 +212,49 @@ function flattenFormElements(
   )
 }
 
+/**
+ * The regex used for matching `{ELEMENT:<elementName>}` tags in OneBlink
+ * Calculation and Info (HTML) elements.
+ */
+const WYSIWYGRegex = /({ELEMENT:)([^}]+)(})/g
```

Comment:
also you'll need to do string.MatchAll to use capture groups with a global flag
```suggestion
const WYSIWYGRegex = /({ELEMENT:)(.*)(})/g
```